docs/overrides.md

```diff
@@ -242,6 +242,52 @@ Variable precedence is as follows:
 1. `uds-config.yaml` variables
 1. Variables `default` in the`uds-bundle.yaml`
 
+#### Variable Types
+Variables can be of either type `raw` or `file`. The type will default to raw if not set explicitly.
+
+> [!WARNING]  
+> If a variable is set to accept a file as it's value, but is missing the `file` type, then the file will not be processed.
+
+```yaml
+kind: UDSBundle
+metadata:
+   name: example-bundle
+   version: 0.0.1
+
+packages:
+   - name: helm-overrides-package
+     path: "../../packages/helm"
+     ref: 0.0.1
+     overrides:
+        podinfo-component:
+          unicorn-podinfo:
+           variables:
+           - name: test_secret
+             path: "testSecret"
+             description: "Key to be set in a secret"
+             type: file
+```
```

Comment:
Love the example! Let's be a bit more explicit about what `raw` vs `file` means. Recommend adding another variable to the list of `variables` and say explicitly what each will resolve to (ie. variable `test-secret` will resolve to the contents of the file `testSecret` <- even better if we make this a more realistic-looking file with an actual extension)

Reply:
added a "side by side" raw example and some descriptions to more explicitly detail what's going on. let me know what you think.